vitamin/src/main/res/values/styles.xml

```diff
@@ -17,5 +17,7 @@
         <item name="buttonGhostReversed">@style/Widget.Vitamin.Button.Ghost.Reversed</item>
         <!--  Checkboxes -->
         <item name="checkboxStyle">@style/Widget.Vitamin.Checkbox</item>
+        <!--  RadioButton -->
+        <item name="radioButtonStyle">@style/Widget.Vitamin.RadioButton</item>
```

Comment:
If we don't use `VitamonRadioButton` but directly `RadioButton`, the component will be customized?

Reply:
Not sure what's the question behind but no, not using vtmn RadioButton style will not customize the component properly. In the sample the radio become yellowish.

Reply:
That could be great to apply the style on the native property (guess `android:radioButtonStyle`?) to have a correct visual for legacy components too. What do you think? It is the work done for buttons.

Reply:
Okay, sounds reasonable to me but I don't see any override for other components. Am I missing something ?